examples/AffinePlaneCurve.jl

```diff
@@ -0,0 +1,124 @@
+module AffinePlaneCurveModule
+
+using Oscar, Markdown
+
+export AffinePlaneCurve, AffinePlaneCurve_degree, AffinePlaneCurve_Jacobian, AffinePlaneCurve_tangent, AffinePlaneCurve_issmoothpoint, AffinePlaneCurve_union
```

Comment:
This seems awfully verbose. These will have names like Oscar.AffinePlaneCurveModule.AffinePlaneCurve_Jacobian
which barely fits on a line.

Is the second AffinePlaneCurve_ actually necessary here?